content/webapp/services/prismic/fetch/index.ts

```diff
@@ -75,13 +75,15 @@ export function fetcher<Document extends PrismicDocument>(
       { client }: GetServerSidePropsPrismicClient,
       id: string
     ): Promise<Document | undefined> => {
-      const document = await client.getByID<Document>(id, {
-        fetchLinks,
-      });
+      try {
+        const document = await client.getByID<Document>(id, {
+          fetchLinks,
+        });
 
-      if (document.type === contentType) {
-        return document;
-      }
+        if (document.type === contentType) {
+          return document;
+        }
+      } catch {}
```

Comment:
This is the same as the `try { … } catch {}` used in `articles.ts`, which already has the correct behaviour.